lib/manager/github-actions/extract.ts

```diff
@@ -1,30 +1,59 @@
+import * as githubTagsDatasource from '../../datasource/github-tags';
 import { logger } from '../../logger';
 import * as dockerVersioning from '../../versioning/docker';
+import * as looseVersioning from '../../versioning/loose';
 import { PackageDependency, PackageFile } from '../common';
 import { getDep } from '../dockerfile/extract';
 
 export function extractPackageFile(content: string): PackageFile | null {
   logger.debug('github-actions.extractPackageFile()');
   const deps: PackageDependency[] = [];
   for (const line of content.split('\n')) {
-    // old github actions syntax will be deprecated on September 30, 2019
-    // after that, the first line can be removed
-    const match =
-      /^\s+uses = "docker:\/\/([^"]+)"\s*$/.exec(line) ||
-      /^\s+uses: docker:\/\/([^"]+)\s*$/.exec(line);
-    if (match) {
-      const [, currentFrom] = match;
+    if (line.trim().startsWith('#')) {
+      continue; // eslint-disable-line no-continue
+    }
+
+    const dockerMatch = /^\s+uses: docker:\/\/([^"]+)\s*$/.exec(line);
+    if (dockerMatch) {
+      const [, currentFrom] = dockerMatch;
       const dep = getDep(currentFrom);
       logger.debug(
         {
           depName: dep.depName,
           currentValue: dep.currentValue,
           currentDigest: dep.currentDigest,
         },
-        'Docker image inside GitHub Actions'
+        'Docker image inside GitHub Workflow'
       );
       dep.versioning = dockerVersioning.id;
       deps.push(dep);
+      continue; // eslint-disable-line no-continue
+    }
+
+    const tagMatch = /^^\s+-?\s+?uses: (?<lookupName>[a-zA-Z-_]+\/[a-zA-Z-_]+)(?<path>.*)?@(?<currentValue>.+?)\s*?$/.exec(
+      line
+    );
+    if (tagMatch) {
+      const { lookupName, path, currentValue } = tagMatch.groups;
+      if (looseVersioning.api.isValid(currentValue)) {
+        const dep: PackageDependency = {
+          depName: `${lookupName}${path || ''}`,
+          lookupName,
+          currentValue,
+          commitMessageTopic: '{{depName}}} action',
+          datasource: githubTagsDatasource.id,
+          versioning: looseVersioning.id,
+          rangeStrategy: 'pin',
+        };
+        logger.debug(
+          {
+            depName: dep.depName,
+            currentValue: dep.currentValue,
```

Comment:
```suggestion
            dep,
```

Reply:
All the other examples were abbreviated, so wasn't sure. Changed

Reply:
ah, i see now. some manager log the dep and some not. 🤔 

@rarkins Should we pefer to remove the logging or should we add those logging to all managers? Maybe log at trace level?

Reply:
No, we ideally shouldn't have that in managers as it bloats the logs. We should convert them to trace